### DISCUSSION TITLE
Integrating HB with Typescript

### DISCUSSION THREAD

**Discussion by sedhuait** [honeybadger-io/honeybadger-js#723] [Q&A] [answered]
Hey there, I was on HB 3.2.7 and I'm trying to update it to v4.0.0-beta.1 and facing the following error. 

Could someone help? Do we have an upgrade guide? 

![image](https://user-images.githubusercontent.com/1628931/160985324-9551c06b-95a7-4d3a-acf8-ba9ebebff28b.png)

**Comment by joshuap** [comment]
cc @subzero10 @shalvah

**Reply by subzero10** [reply]
Thanks! I will look at this asap!

**Comment by subzero10** [comment, marked as answer]
@sedhuait This has been fixed with the latest beta release of v4 (v4.0.0-beta.4). Let us know if you have any other issues. Thanks!
